multi-node/aws/pkg/config/templates/cluster.yaml

```diff
@@ -72,6 +72,31 @@ kmsKeyArn: "{{.KMSKeyARN}}"
 # Price (Dollars) to bid for spot instances. Omit for on-demand instances.
 # workerSpotPrice: "0.05"
 
+## Etcd Cluster config
+## WARNING: Any changes to etcd parameters after the cluster is first created will not be applied
+## during a cluster upgrade, due to concerns over data loss.
+## This situation is being rectified with work towards automated management of etcd clusters
+
+# Number of etcd nodes
+# (Set to an odd number >= 3 for HA control plane)
+# etcdCount: 1
```

Comment:
Will this also enable changing the cloud-config-* files and redeploying the changes through a new compiled CloudFormation template? If so, then this is awesome :)